Integration_App3/Part3_App3/src/WeatherDataSubject.java

```diff
@@ -0,0 +1,77 @@
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class WeatherDataSubject {

```

Comment:
WeatherDataSubject, you had to create a WeatherDataSubject interface

Reply:
I have to make it an interface for :-
Because it is expected that in the future, when developing the program, more types of subjects will be added.Therefore, it is necessary to create an interface for the fundamental functions (  functions found in any subject that is constructed) so that subjects can implement it.
Am I understanding correctly?